src/views/virtualmachinesinstance/details/tabs/disks/hooks/useFileSystemTableGuestOS.tsx

```diff
@@ -28,6 +28,8 @@ const useFileSystemTableGuestOS: UseFileSystemTableGuestOS = (vmi) => {
         setData(jsonData);
         setLoaded(true);
       })().catch((err) => setError(err));
+    } else if (!guestOS) {
+      setLoaded(true);
```

Comment:
check if working ok on windows machines and F5

Reply:
removed